notebooks/transport/http.go

```diff
@@ -5,12 +5,13 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/influxdata/influxdb/v2"
 	feature "github.com/influxdata/influxdb/v2/kit/feature"
 	"github.com/influxdata/influxdb/v2/kit/platform"
 	"github.com/influxdata/influxdb/v2/kit/platform/errors"
 	kithttp "github.com/influxdata/influxdb/v2/kit/transport/http"
 
+	notebooks "github.com/influxdata/influxdb/v2/notebooks/service"
+
```

Comment:
Extra whitespace here (will probably need reformatted once it's deleted)

Reply:
Took care of it, thanks!